.github/dependabot.yml

```diff
@@ -3,6 +3,10 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: weekly
+    interval: daily
```

Comment:
I set it to weekly to avoid spamming. Any special reason that you want to make it shorter?

Reply:
No particular reason.
I restored to weekly.